README.md

```diff
@@ -62,7 +62,7 @@ djenriquez/vault-ui
 ```
 
 Supported environment variables:
-- `NODE_TLS_REJECT_UNAUTHORIZED` disable TLS server side validation (ex. vault deployed with self-signed certificate)
+- `NODE_TLS_REJECT_UNAUTHORIZED` 0 or 1 (default: 0), disable TLS server side validation (ex. vault deployed with self-signed certificate)
```

Comment:
I think you're correct, it's not about the value of `NODE_TLS_REJECT_UNAUTHORIZED`. it's about it being set or unset